_posts/2020-12-17-solidity-v0.8.0-release-announcement.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.0 Release Announcement'
+date: '2020-12-17'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity 0.8.0](https://github.com/ethereum/solidity/releases/tag/v0.8.0) is a breaking release of the Solidity compiler and language.
+
+This release does not include many features but rather changes that require a
+backwards-incompatible adjustment in syntax or semantics. For a detailed explanation,
+please see the [documentation](https://docs.soliditylang.org/en/latest/080-breaking-changes.html).
+
+The change that will affect most users is that arithmetic operations are now checked by default,
+which means that overflow and underflow will cause a revert. This feature can be disabled locally.
```

Comment:
The first question that popped into my mind when I read this was "how" so I think I'd add this:

```suggestion
which means that overflow and underflow will cause a revert. This feature can be disabled locally by using an ``unchecked`` block.
```